pkg/apis/extensions/v1beta1/types.go

```diff
@@ -967,10 +967,11 @@ type PodSecurityPolicySpec struct {
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty" protobuf:"varint,14,opt,name=readOnlyRootFilesystem"`
 	// SeccompProfiles lists the allowed profiles that may be set for the pod or
-	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
-	// be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
-	// used to generate a value for a pod the first non-wildcard profile will be used as
-	// the default.
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles
+	// are allowed to be set and that a default (docker/default) will be set by the strategies.
+	// A non-empty value contains the list of allowed profiles and the first non-wildcard (*) should
```

Comment:
I agree.  This allows the container runtime itself or the container runtime code in the kubelet to provide anything it wants for nil settings without the PSP having to be aware of the runtime to set default values.  If values are actually set in the PSP the admin is setting them and is already aware of the runtime being used. I think a mixed runtime environment is a corner case that we probably don't need to worry about which seems like the only sticky point.

Will revert the feedback commits - the original commits which allowed nils seems like the right way to go.